cpanfile

```diff
@@ -29,7 +29,7 @@ requires 'Moose::Role';
 requires 'Moose::Util::TypeConstraints';
 requires 'MooseX::NonMoose';
 requires 'Number::Format';
-requires 'PGObject', '1.403.2';
+requires 'PGObject', '< 2'; # '1.403.2';
```

Comment:
If you need these to be numbers, then `1.403.2` is represented as `1.403002` so, we could have `>=1.403002 <2` which is a better representation of our requirements than simply smaller than 2, I'd say. Could you try that?